libtransmission/rpcimpl.c

```diff
@@ -632,6 +641,10 @@ static void addField(tr_torrent* const tor, tr_info const* const inf, tr_stat co
         tr_variantDictAddBool(d, key, st->isStalled);
         break;
 
+    case TR_KEY_labels:
```

Comment:
the new getter / setter RPC API needs to be documented in `extras/rpc-spec.txt`, both in the torrent get/set section and in the changelog at the bottom